README.md

```diff
@@ -126,16 +126,9 @@ queue.post(
 
 **Optional messages' parameters:**
 
-* `timeout`: After timeout (in seconds), item will be placed back onto queue.
-You must delete the message from the queue to ensure it does not go back onto the queue.
- Default is 60 seconds. Minimum is 30 seconds. Maximum is 86,400 seconds (24 hours).
-
 * `delay`: The item will not be available on the queue until this many seconds have passed.
 Default is 0 seconds. Maximum is 604,800 seconds (7 days).
 
```

Comment:
Can you add 'push_headers' to the optional parameters? See here: http://dev.iron.io/mq/3/reference/api/#post-messages